packages/ui/src/kit/VInput/VInput.theme-default.scss

```diff
@@ -0,0 +1,38 @@
+.v-input.theme-default .v-input-reset-button {
+  line-height: 0;
```

Comment:
are you sure about set `line-height: 0`?

Reply:
and as i said, that you need use `<button>` tag with svg-atom, instead of `icon-button-molecule`
because you set size of `icon-button-molecule` to 14/14px and this means that button will have the same dimesion and this is too small for clicks.

or maybe you need change icon-button-molecule to be `<button>` with icon inside and

Reply:
If I'd using `button`, I'd set size `14px` as well, and it will have same dimensions as well. I'd doing my best to make same with `icon-button-molecule` - according to design. So it will no difference.

Do you suggest add some area (in 2-3px) around icon-button to make clickable area more? If yes, you should increase size of the button, because it's looks very strange, when you hover cursor on empty space, but element near react with hovered state.